librubyfmt/src/format.rs

```diff
@@ -889,22 +889,37 @@ pub fn format_kw(ps: &mut dyn ConcreteParserState, kw: Kw) {
     }
 }
 
+pub fn format_backtick(ps: &mut dyn ConcreteParserState, backtick: Backtick) {
+    if ps.at_start_of_line() {
+        ps.emit_indent();
+    }
+
+    handle_string_and_linecol(ps, backtick.1, backtick.2);
+
+    if ps.at_start_of_line() {
+        ps.emit_newline();
+    }
+}
```

Comment:
This function has been copied at least 4 times handling different types and should get refactored.